sdk/dotnet/Kms/Registry.cs

```diff
@@ -9,63 +9,32 @@
 
 namespace Pulumi.Gcp.Kms
 {
-    /// <summary>
-    ///  Creates a device registry in Google's Cloud IoT Core platform. For more information see
-    /// [the official documentation](https://cloud.google.com/iot/docs/) and
-    /// [API](https://cloud.google.com/iot/docs/reference/cloudiot/rest/v1/projects.locations.registries).
-    /// </summary>
     public partial class Registry : Pulumi.CustomResource
```

Comment:
Don't we mark renamed resources as `[Obsolete]`?

Reply:
We do not as of this moment - it's something you added an issue for in the bridge https://github.com/pulumi/pulumi-terraform-bridge/issues/133

Reply:
Sorry, I keep forgetting. We should prioritize that task.